Update quick start page (#946)
meilisearch/documentation: learn/getting_started/quick_start.md

There are [several ways to let MeiliSearch know what the primary key](/learn/core_concepts/documents.md#primary-key) is. The easiest one is to have an <clientGlossary word="attribute" /> that contains the string `id` in a case-insensitive manner.

> Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.

Author: ```suggestion
Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right-click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.
```

> Below is an example to showcase how to add documents to an index called `movies` using the following test dataset: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Right-click the movies.json file link and move the downloaded file to your working directory.

Reviewer: ```suggestion
Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.
```
